TezosKit/Common/Michelson/DecimalMichelsonParameter.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  DecimalMichelsonParameter.swift
+//  TezosKit
+//
+//  Created by Simon Mcloughlin on 17/02/2020.
+//
+
+import Foundation
+
+/// A representation of an Decimal parameter in Michelson.
+public class DecimalMichelsonParameter: AbstractMichelsonParameter {
```

Comment:
Totally cool with using decimals. 

These parameters map (AFAICT) 1:1 with the Micheline JSON language: https://tezos.gitlab.io/whitedoc/michelson.html#json-syntax

There's no decimal paramter, so I'd favor adding this to the IntMichelsonParameter as an alternative initializer. 

Aside: have you figured out if this works? I think that the doc shows `int` parameters with quotes, but in practice I think I found errors if I sent a quoted number. If this is not the case, then we can easily make an API that uses BigIntegers.

Reply:
Ah didn't know that was following a strict standard. Sure made that change.

Yeah I did test it. I've had those screens working in the app for ~2 weeks now. I've been pointing my cocoapods spec to this branch and testing it